pkgs/development/tools/build-managers/goredo/default.nix

```diff
@@ -22,12 +22,15 @@ buildGoModule rec {
 
   nativeCheckInputs = lib.optionals doCheck [ python3 perl ];
 
-  SHARNESS_TEST_SRCDIR = sharness + "/share/sharness";
+  inherit (sharness) SHARNESS_TEST_SRCDIR;
 
   vendorSha256 = null;
   subPackages = [ "." ];
 
-  preBuild = "cd src";
+  preBuild = ''
+    export GOPATH=$TMPDIR/go/bin
```

Comment:
That should already exist. What problem did you try to solve by setting GOPATH?

Reply:
It does not exist. It used to but is not set anymore. Without setting GOPATH, the buildPhase just does nothing.

I am not a go packaging expert, but I suspect that the packages does not adhere to the packaging standards. Setting GOPATH again (like it used to) solves the issue.

Reply:
It is set here https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/go/module.nix#L107

Reply:
Thanks! I don't know how I could miss that... Now, that I'm using it right, it's even simpler than before.